src/common/components/header/index.js

```diff
@@ -7,7 +7,7 @@ import styles from './header.css';
 
 const wordmark = 'https://assets.ubuntu.com/v1/d45097a4-snapcraft.io-logotype.svg';
 
-class Header extends Component {
+export class Header extends Component {
```

Comment:
I'm not sure about moving binding up the tree. This will mean that top level component/container binds the data for all the child components and data is passed down the tree using props. 

Why should parent component bind user and auth if it's only header that needs that data?

There are benefits of both approaches obviously. Binding close to the component that needs the data helps to avoid passing too much data through props. But on the other hand we end up with many components bound to the store.

When it comes to testing and exporting named class.
Yes, it would be possible to remove binding from `Header` and move it up. Actually - that's how it was from the beginning. Now it seems to me that `connect` was only added here to make `dispatch` available (to dispatch logout action). Also, `Header` is rendered only in one place (app container), so it wouldn't be such an issue to pass props to it (as it is done already).

But issue will remain with other components, like `UserAvatar` here.
It's useful for user avatar to have its own binding to the store. Parent component doesn't need to worry about what data UserAvatar needs and why.
In cases of such components, that are bound to the store anyway - we need a way to test them without the need for the store. And pattern with exporting both default bound component and named pure component is very helpful for that.